haiku-apps/haiqr/haiqr-2.0.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="QR generator for Haiku"
+DESCRIPTION="HaiQR is a QR code generator. \
+It's a native python3 program that uses Haiku APIs through Haiku-PyAPI \
+wrapper, pillow and qrcode modules. \
+It allows you create a personalized QR code with a small thumbnail in the \
+middle (if you'd like). You can save the QR generated. \
+The code is released under MIT license."
+HOMEPAGE="https://github.com/tmtfx/HaiQR"
+COPYRIGHT="2024 Fabio Tomat (TmTFx)"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/tmtfx/HaiQR/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="3c0fcc7b7c63b9d9440661d18eac5d623e8bcc5dcb604ca16fc10a222c43245a"
+SOURCE_DIR="HaiQR-$portVersion"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	haiqr = $portVersion
+	app:HaiQR = $portVersion
+	cmd:haiqr.py
+	"
+
+REQUIRES="
+	haiku
+	haiku_pyapi_python310
+	pillow_python310
+	qrcode_python310
```

Comment:
@tmtfx given that HaiQR only uses qrcode_python310, maybe it's good to drop python3.9 for qrcode, given nothing else will use it?

Reply:
You can just delete this file in this PR, should clear up on things, and then create a seperate PR for HaiQR.

Reply:
Mh this pushed these commits along with HaiQR to the [new PR ](https://github.com/haikuports/haikuports/pull/10914)